src/js/edu-benefits/components/SchoolSelectionFields.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+import ErrorableTextarea from '../../common/components/form-elements/ErrorableTextarea';
+import ErrorableTextInput from '../../common/components/form-elements/ErrorableTextInput';
+import ErrorableSelect from '../../common/components/form-elements/ErrorableSelect';
+import DateInput from '../../common/components/form-elements/DateInput';
+import Address from '../../common/components/questions/Address';
+
+import { validateIfDirtyDateObj, isValidDateField } from '../utils/validations';
+import { schoolTypes } from '../utils/options-for-select';
+
+function showSchoolAddress(educationType) {
+  return educationType === 'college'
+    || educationType === 'flightTraining'
+    || educationType === 'apprenticeship'
+    || educationType === 'correspondence';
+}
+
+export default class SchoolSelectionFields extends React.Component {
+  render() {
+    return (<fieldset>
+      <div className="input-section">
+        <ErrorableSelect
+            label="Type of education or training"
+            name="educationType"
+            options={schoolTypes}
+            value={this.props.data.educationType}
+            onValueChange={(update) => {this.props.onStateChange('educationType', update);}}/>
+      </div>
+      <div className="input-section">
+          {showSchoolAddress(this.props.data.educationType.value)
+          ? <div>
+            <p>Do you know what school you will be attending?</p>
```

Comment:
Not a blocker for getting this merged in (might be fastest to review all these things once we have the complete form), but just wanted to point it out to return to later.